frontend/src/lib/components/instances/TableView.svelte

```diff
@@ -175,7 +175,7 @@
 							{#if instanceHidden}
 								<p class="text-center">...</p>
 							{:else}
-								<div class="instance">
+								<div class="min-w-small-instance">
```

Comment:
Would remove the div and move the sizing to the instance view itself.

Reply:
The width in the instance view changes though depending on what view it's in, e.g. reports